### DISCUSSION TITLE
amis 交流群

### DISCUSSION THREAD

**Discussion by xinlanlan** [baidu/amis#5322] [General]
最近在使用webpack基于amis-editor进行搭建低代码平台，希望有问题可以一起交流。
我的微信: flytiger123
群：
<img width="312" alt="image" src="https://user-images.githubusercontent.com/23093206/188572816-b5b285ed-8158-4057-8a6c-61583752d278.png">

**Comment by enjoy-wind** [comment]
还有其他群吗
